heron_base/config/imu_compass.yaml

```diff
@@ -1,13 +1,6 @@
-# Default configuration. To achieve more accuracy in the compass, periodically
-# regenerate. With Heron deployed in water, run
-#
-#   rosrun heron_bringup calibrate_compass
-#
-# Heron will rotate slowly for 60 seconds and a new calibration will be
-# generated.
-#
+# Generated from /tmp/calibrate_compass.KtxH/imu_record.bag on 2018-05-07.
```

Comment:
Revert to the old file with the comment on how to configure the compass calibration.